PEP 656: Clarification on platform definition and musl detection (#1877)
python/peps: pep-0656.rst

require wheels using this tag make similar promises. Please refer to
the PEP for more details on rationale and reasoning behind the design.

The ``musllinux`` platform tags only apply to Python interpreters

> dynamically linked aginst the musl libc and executed on the runtime

Reviewer: ```suggestion
dynamically linked against the musl libc and executed on the runtime
```

    musllinux_${MUSLMAJOR}_${MUSLMINOR}_${ARCH}

Reading the musl version
------------------------

> The musl version values can be obtains by executing the musl libc

Reviewer 2: ```suggestion
The musl version values can be obtained by executing the musl libc
```

shared library the Python interpreter is currently running on, and
parsing the output::

    import re
    import subprocess

    def get_musl_major_minor(so: str) -> tuple[int, int] | None:
        """Detect musl runtime version.

        Returns a two-tuple ``(major, minor)`` that indicates musl

>         library's version, or ``None`` if the given value does not

Reviewer: ```suggestion
        library's version, or ``None`` if the given libc .so does not
```

:+1: for providing this function btw :)

>         library's version, or ``None`` if the given lib .so does not

Reviewer: Ah my bad, I made a typo in my suggestion :facepalm: 
```suggestion
        library's version, or ``None`` if the given libc .so does not
```

Author: I amended the last commit 🙂

Author: I’ve amended the last commit 🙂
